corehq/apps/export/templates/export/spec/ko/mocha.html

```diff
@@ -10,6 +10,5 @@
 {% block mocha_tests %}
 <script src="{% static 'export/spec/data/export_instances.js' %}"></script>
 <script src="{% static 'export/spec/ExportInstance.spec.js' %}"></script>
-<script src="{% static 'export/spec/ExportColumn.spec.js' %}"></script>
```

Comment:
huh that is interesting. was it failing when running from the console or the browser?